src/MakersOfDenmark.Domain/Models/MakerSpace.cs

```diff
@@ -16,6 +18,15 @@ public class MakerSpace : Entity<Guid>
         public AccessType AccessType { get; set; }
         public Organization Organization { get; set; }
         public ICollection<Tool> Tools { get; set; } = new List<Tool>();
-
+        public ICollection<MODUserFollowMakerSpace> Followers { get; set; } = new List<MODUserFollowMakerSpace>();
```

Comment:
Egentligt så burde dette være et HashSet. På den måde sikre vi os at der ikke kan være nogle duplicate followers på datastruktur niveau,

https://docs.microsoft.com/en-us/dotnet/api/system.collections.generic.hashset-1?view=netcore-3.1

Det samme kunne argument kan bruges omkring Tools.

Reply:
Jeg syntes godt vi kan kigge på Hashset<>.
Jeg har aldrig arbejdet direkte med dem,
Kræver det at vi laver noget override GetHashCode() på de klasser vi vil smide ind i Hashset?

Reply:
Det kræver at vi overrider GetHashCode, Equals og == != på den Entitet vi smider ind for at have ens opførsel.

Disse overrides kan vi blot have på Entity<T>